include/seqan3/search/algorithm/configuration/max_error.hpp

```diff
@@ -122,10 +122,13 @@ struct search_config_max_error_adaptor : public configuration_fn_base<search_con
         // at least one specific error type specified
         else if constexpr (!total_set)
         {
+            uint8_t substitution_error = static_cast<uint8_t>(std::get<1>(out));
```

Comment:
Can you make all three `const`?

Reply:
```suggestion
            uint8_t const substitution_error = static_cast<uint8_t>(std::get<1>(out));
            uint8_t const insertion_error = static_cast<uint8_t>(std::get<2>(out));
            uint8_t const deletion_error = static_cast<uint8_t>(std::get<3>(out));
```

(not sure whether multi-line suggestions work)

Reply:
will change! :+1: